app/src/debug/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_launcher_name">Camera_debug</string>
+    <!--TODO Revert to Camera_debug -->
+    <string name="app_launcher_name">CameraX_debug</string>
```

Comment:
To facilitate having both the older app and the one with Camera X on the same device, I changed the debug app name. It would be reverted.